app/models/mortgage_calculator/affordability.rb

```diff
@@ -251,5 +253,16 @@ def annual_committed_costs
     def default_borrowing_amount
       (can_borrow_from + can_borrow_upto) / 2
     end
+
+    private
+
+    def borrowing=(value)
+      @borrowing = unformat(value)
+    end
+
+    def unformat(value)
```

Comment:
Breaking this out into a separate method doesn't seem necessary given it's only used by one method, and they're both tiny.

How about:

```
def borrowing=(value)
  @borrowing = value.is_a?(String) ? value.gsub(/,/, '') : value
end
```

Also avoids the guard condition which I'm not keen on, especially when combined with `unless`  (just personal taste though).